test/e2e_node/memory_manager_test.go

```diff
@@ -802,7 +936,757 @@ var _ = SIGDescribe("Memory Manager Incompatibility Pod Level Resources", framew
 				initCtnParams = []memoryManagerCtnAttributes{}
 			})
 
-			ginkgo.It("should not report any memory data during request to pod resources List when pod has pod level resources", func(ctx context.Context) {
+			ginkgo.It("scope: pod, should allocate exclusive memory to a guaranteed pod with pod-level resources and guaranteed container, PodLevelResourceManagers enabled", func(ctx context.Context) {
```

Comment:
(Very) nice to have, not blocking: instead of using `scope: pod` in the `It` description, it would be better to use ginkgo labels.

Reply:
Done.